app/controllers/api/v1/ratings_controller.rb

```diff
@@ -1,18 +1,15 @@
 module Api
   module V1
     class RatingsController < ApplicationController
-      before_action :set_rating, only: %i[show update destroy]
+      before_action :authenticate_user!
```

Comment:
I think you can put it to the ApplicationController

Reply:
are you sure about put authenticate_user in application controller? i think guest user can see movies and persons without registration

Reply:
we can add skip_before_action for movie and person controllers